<launch>
    <!-- Subscribed topics -->
    <arg name="video_path"  default="./save.avi" />
    <arg name="txt_path"    default="./save.avi.txt"/>

    <node pkg="video_recorder" type="video_decorder" name="video_decorder"  output="screen" required="true" clear_params="true">
        <param name="path" type="string" value= "$(arg video_path)" />
        <param name="txt_path" type="string" value="$(arg txt_path)" />
        <param name="frame_id" type="string" value="rgb_camera_link" />
        <param name="topic" type="string" value="/rgb/image_raw" />
    </node>
</launch>